### PR TITLE
Added playerfish invincibility

### PR DESCRIPTION
**Pull request by Taeir**
Playerfish can be invincible until a certain timestamp.
Single Player Tests are more stable, because fish cannot die in them.

<feature,testing

### CODE REVIEW

**Comment by jessearens** [comment]
Even though the commits seem to look fine to me, the build does not pass. Therefore I'll close the pull request and let you have a look at the reason why the build does not pass.

<bug

**Comment by Taeir** [comment]
Fixed failing tests again

<bug,testing

**Comment by Diocruel** [comment]
.Woha, that invincibility is soooooo handy. +1

<feature,testing

File: src/main/java/com/github/fishio/control/SinglePlayerController.java [hunk 1]
```diff
@@ -255,6 +255,25 @@ public void restartGame() {
 	}
 	
 	/**
+	 * Updates the pause button to the correct state.
+	 */
+	public void updatePauseButton() {
```
Comment (ChielBruin) [inline src/main/java/com/github/fishio/control/SinglePlayerController.java:260]: Nice addition!
<feature

**Comment by ChielBruin** [comment]
Hopefully this is the final fix for the gui tests!
Good job :+1: 
<bug

**Pull request merged**